src/content/blog/2024_09_aquapointer.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Aquapointer, a software package for quantum biology applications" 
+author: Misty Wahl
+day: 6
+month: 9
+year: 2024
+tags: 
+  - Q4Bio
+  - Open source software
+---
```

Comment:
I replaced OSS with "Research". Not sure about grants because most of our research is grant-supported, and it could be confused with the microgrant program.